docs/shared-content-source/docs/modules/develop/pages/akka-streamlet-utilities.adoc

```diff
@@ -6,6 +6,7 @@ include::ROOT:partial$include.adoc[]
 The `cloudflow.akka.util` library contains some predefined `StreamletLogic`{empty}s:
 
 - `HttpServerLogic`
+- `GrpcServerLogic`
```

Comment:
Perhaps we should move `HttpServerLogic` and `GrpcServerLogic` to their own pages, WDYT?

Reply:
Yes I think that is a good idea.

Reply:
yes good idea